src/api/storage-volumes.tsx

```diff
@@ -287,8 +288,10 @@ export const migrateStorageVolume = async (
         "Content-Type": "application/json",
       },
       body: JSON.stringify({
+        project: targetProject,
+        migration: Boolean(target),
         name: volume.name,
-        pool: targetPool,
+        pool: targetPool ?? volume.pool,
       }),
```

Comment:
Somehow, this doesn't work for cluster member migration. I am not sure what the correct payload is to trigger it correctly. Hope we get some hints on how to initiate this.